specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2018-02-01-preview/examples/ContainerGroupsUpdate.json

```diff
@@ -0,0 +1,32 @@
+{
+    "parameters": {
+      "subscriptionId": "subid",
+      "api-version": "2017-12-01-preview",
+      "resourceGroupName": "demoResource",
+      "containerGroupName": "demo1",
+      "Resource": {
+        "id": "/subscriptions/subid/resourceGroups/demoResource/providers/Microsoft.ContainerInstance/containerGroups/demo1",
+        "location": "WestUs",
+        "name": "demo1",
+        "type": "Microsoft.ContainerInstance/resource",
+        "tags": {
+          "tag1key": "tag1Value",
+          "tag2key": "tag2Value"
+        }
+      }
+    },
+    "responses": {
+      "200": {
+        "body": {
+          "id": "/subscriptions/subid/resourceGroups/demoResource/providers/Microsoft.ContainerInstance/containerGroups/demo1",
```

Comment:
Sorry I just caught this now - but the `properties` property bag defined in `ContainerGroup` is missing in the response.

Reply:
@jianghaolu fixed. Please take a look.